### ISSUE TITLE
Should urls be read?

### ISSUE THREAD

**Issue by sakhdevel** [joelpurra/talkie#35]
_Please fill out the following to help us help you. Replace `...` with your own values, where applicable._

It is not very useful and clear-to-listen when extension reads full urls.

...

Don't read them at all or say "link" or prounounce domain part of url or make reading them optional

...

### Steps to reproduce behavior

...

### Website

any web site

### Text and language

- Which part of the website did you want spoken: ...
- If possible, include the full text: ...
- Language the text was expected to be spoken in: ...

### System information

- Your browser: ...
- Your browser version: ...
- Your operating system: ...
- Your operating system version: ...

### Additional information

...

**Comment by joelpurra** [comment]
@sakhdevel: yes, URL shortening sounds like a good idea. Perhaps "link to example.com". Ideally it would be configurable, but would probably start with a simple enable/disable option. Unfortunately I cannot say if/when this would be implemented.
